app/waveplot/slater.F90

```diff
@@ -7,55 +7,66 @@
 
 #:include 'common.fypp'
 
-!> Routines to calculate a Slater type orbital (STO)
+!> Routines to calculate a Slater type orbital (STO).
 module waveplot_slater
   use dftbp_common_accuracy, only :dp
   implicit none
 
   private
   save
 
+  public :: realTessY
+  public :: TSlaterOrbital, TSlaterOrbital_init, getValue, assignment(=)
 
-  !> Data for STOs
+
+  !> Data type for STOs.
   type TSlaterOrbital
     private
+
+    !>
```

Comment:
Most of this type's component comments match the variables in the TSlaterOrbital_getValue_explicit subroutine, so could be copied over. Those comments aren't completely clear, but are better than nothing. Then gridValue is the radial grid and gridDist the spacing and nGrid the number of points (there is reduncancy between these quantities, but it's how it is coded).

Reply:
You're right, I've copied over/added the missing comments.